doc/changes/latest.inc.template

```diff
@@ -18,7 +18,7 @@
 
 .. _current:
 
-Current (1.1.dev0)
+Current (1.2.dev0)
```

Comment:
Is this on purpose? Looks dangerous to me.

Reply:
This was just from rebasing and being the first entry and so I changed the latest file and copied it over. It's just cruft now, I'll be sure it's not touched when the PR is ready to merge and not a work in progress